src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/OptionalReferenceInputArgumentCollection.java

```diff
@@ -11,10 +11,21 @@
 public final class OptionalReferenceInputArgumentCollection extends ReferenceInputArgumentCollection {
 
     @Argument(fullName = StandardArgumentDefinitions.REFERENCE_LONG_NAME, shortName = StandardArgumentDefinitions.REFERENCE_SHORT_NAME, doc = "Reference sequence", optional = true)
-    private File referenceFile;
+    private String referenceFileName;
 
+    private File referenceFile = null;
+
+    // synchronized to avoid a race in the check-then-modify sequence
     @Override
-    public File getReferenceFile() {
+    public synchronized File getReferenceFile() {
```

Comment:
Since hellbender is strictly single-threaded, I don't see a need for synchronization here.